activerecord/attibute.go

```diff
@@ -0,0 +1,157 @@
+package activerecord
+
+import (
+	"fmt"
+)
+
+const (
+	Int    = "int"
+	String = "string"
+)
+
+type Attribute interface {
+	AttributeName() string
+	IsPrimaryKey() bool
```

Comment:
Would it be better to define the `PrimaryKey` attribute, that replies to `interface { IsPrimaryKey() bool }`?